packages/components/src/post-locked-modal/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+import { PostPreviewButton, getWPAdminURL } from '@wordpress/editor';
+import './style.scss';
+import { addQueryArgs } from '@wordpress/url';
+import { getPostEditURL } from '../../../../edit-post/components/browser-url';
```

Comment:
Not certain why we need this function from the `edit-post` module? Can't we ust the `@wordpress/url` module instead.

Reply:
sure, does that provide the post edit url? I didn't realize that was available.

Reply:
I'm not certain yet, I have to dig more in the code :)